naming/src/main/java/com/alibaba/nacos/naming/remote/rpc/handler/InstanceRequestHandler.java

```diff
@@ -46,6 +51,10 @@ public InstanceRequestHandler(EphemeralClientOperationServiceImpl clientOperatio
     @Override
     @Secured(action = ActionTypes.WRITE, parser = NamingResourceParser.class)
     public InstanceResponse handle(InstanceRequest request, RequestMeta meta) throws NacosException {
+        if (!upgradeJudgement.isUseGrpcFeatures()) {
+            throw new NacosException(NacosException.SERVER_ERROR,
+                    "Nacos server can not use gRPC features now, please try again later.");
```

Comment:
The hint can change to `Nacos cluster is running with 1.X mode, can't accept gRPC request temporarily. Please check the server status or close Double write to force open 2.0 mode. Detail https://nacos.io/en-us/docs/2.0.0-upgrading.html`

Reply:
The hint has been changed.